src/prefect/agent/agent.py

```diff
@@ -189,11 +195,13 @@ def query_flow_runs(self, tenant_id: str) -> list:
         }
 
         now = pendulum.now("UTC")
+        self.logger.debug("Current process time {}".format(now.isoformat()))
```

Comment:
I'm happy to keep this in, but note that the timestamps will be included in the logging output as well.